shipment_advice_planner/wizards/shipment_advice_planner.py

```diff
@@ -0,0 +1,151 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import Command, _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class ShipmentAdvicePlanner(models.TransientModel):
+    _name = "shipment.advice.planner"
+    _description = "Shipment Advice Planner"
+
+    picking_to_plan_ids = fields.Many2many(
+        comodel_name="stock.picking",
+        string="Pickings to plan",
+        required=True,
+        domain='[("can_be_planned_in_shipment_advice", "=", True),'
+        '("warehouse_id", "=?", warehouse_id),]',
```

Comment:
Take outgoing picking types of the warehouse instead of relying on the warehouse on the picking